src/NuGet.Clients/NuGet.VisualStudio.Common/IDE/VsMonitorSelectionExtensions.cs

```diff
@@ -34,10 +35,28 @@ public static EnvDTE.Project GetActiveProject(this IVsMonitorSelection vsMonitor
                 // multiple items are selected.
                 if (pitemid == (uint)VSConstants.VSITEMID.Selection)
                 {
-                    return null;
+                    uint numberOfSelectedItems;
+                    int isSingleHierarchyInt;
+                    if (ErrorHandler.Succeeded(ppMIS.GetSelectionInfo(out numberOfSelectedItems, out isSingleHierarchyInt)))
+                    {
+                        bool isSingleHierarchy = (isSingleHierarchyInt != 0);
```

Comment:
If this is `false`, avoid the array allocation and call to `GetSelectedItems`. `hierarchy` will not be assigned anyway.